### PR TITLE
adding mexican-mule recipe

### PR DESCRIPTION
**Pull request by flyfishingbarbara**

### CODE REVIEW

File: src/recipes/mexican-mule.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+        {
+            "quantity": "6",
+            "measure": "oz",
+            "ingredient": "Ginger Beer"
+        },
+         {
+            "quantity": "4",
+            "measure": "dashes",
+            "ingredient": "Angostura Bitters"
+        },
```
Comment (alfg) [inline src/recipes/mexican-mule.json]:
```suggestion
        }
```

File: src/recipes/mexican-mule.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+        "Pour Tequila, fresh lime juice, bitters and Ginger beer over ice in glass or copper mug",
+        "Stir and garnish with lime wedges",
+        "Enjoy!"
+    ],
+    "image": "mexican-mule.jpg",
+    "keywords": [
+        "Tequila",
+        "Mule",
+        "Alcoholic",
+        "Ginger Beer",
```
Comment (alfg) [inline src/recipes/mexican-mule.json]:
```suggestion
        "Ginger Beer"
```

Comment (alfg) [inline src/recipes/mexican-mule.json]: All lowercase for keywords, please.

**Comment by flyfishingbarbara** [comment]
Thanks! Will make those changes...

**Comment by alfg** [comment]
Thanks!

**Pull request merged**